### DISCUSSION TITLE
Custom Serverless Routes?

### DISCUSSION THREAD

**Discussion by helloimalastair** [vercel/vercel#5263] [Help] [answered]
Is there any way to remap the serverless functions to a different route, such as the root of the domain, instead of root/api/example, or is this hard-baked into the system?

**Comment by mcsdevv** [comment, marked as answer]
Hello @goalastair, yep, you can do that with `rewrites`: https://vercel.com/docs/configuration#project/rewrites

Or, if using Next.js: https://nextjs.org/docs/api-reference/next.config.js/rewrites

**Reply by helloimalastair** [reply]
Thank you so much!
